scripts/installscripts/buster-install-default.sh

```diff
@@ -467,11 +467,14 @@ esac
 # power management of wifi: switch off to avoid disconnecting
 sudo iwconfig wlan0 power off
 
+# create backup of /etc/resolv.conf
+sudo cp /etc/resolv.conf /etc/resolv.conf.orig
+
 # Generate locales
 sudo locale-gen "${LANG}"
 
 # Install required packages
-
+sudo apt-get --yes --allow-downgrades --allow-remove-essential --allow-change-held-packages install apt-transport-https
```

Comment:
I got a note here: `Note, selecting 'apt' instead of 'apt-transport-https'`.

Don't know, if this causes any issues.